consensus/parlia/parlia.go

```diff
@@ -1065,6 +1111,42 @@ func (p *Parlia) verifyValidators(header *types.Header) error {
 	return nil
 }
 
+func (p *Parlia) verifyTurnTerm(chain consensus.ChainHeaderReader, header *types.Header) error {
+	if header.Number.Uint64()%p.config.Epoch != 0 ||
+		!p.chainConfig.IsBohr(header.Number, header.Time) {
+		return nil
+	}
+
+	turnTermFromHeader, err := parseTurnTerm(header, p.chainConfig, p.config)
+	if err != nil {
+		return err
+	}
+	if turnTermFromHeader != nil {
+		parent := chain.GetHeaderByHash(header.ParentHash)
+		if parent == nil {
+			return errors.New("parent not found")
+		}
+
+		if p.chainConfig.IsBohr(parent.Number, parent.Time) {
+			turnTermFromContract, err := p.getTurnTerm(parent)
+			if err != nil {
+				return err
+			}
+			if turnTermFromContract != nil && uint8(turnTermFromContract.Int64()) == *turnTermFromHeader {
+				log.Debug("verifyTurnTerm", "turnTerm", turnTermFromContract.Int64())
+				return nil
+			}
+		} else {
+			if uint8(defaultTurnTerm) == *turnTermFromHeader {
```

Comment:
Same as before, this line cannot be reached, because the previous IsBohr check has returned.

Reply:
> Same as before, this line cannot be reached, because the previous IsBohr check has returned.
☝️

Reply:
> > Same as before, this line cannot be reached, because the previous IsBohr check has returned.
> > refer above reply from @buddh0  ☝️